Command/ImportMappingDoctrineCommand.php

```diff
@@ -29,6 +29,8 @@ protected function configure()
             ->addOption('shard', null, InputOption::VALUE_REQUIRED, 'The shard connection to use for this command')
             ->addOption('filter', null, InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY, 'A string pattern used to match entities that should be mapped.')
             ->addOption('force', null, InputOption::VALUE_NONE, 'Force to overwrite existing mapping files.')
+            ->addOption('bundle-less', null, InputOption::VALUE_NONE, 'Bundle argument will be used as destination path')
+            ->addOption('root-namespace', null, InputOption::VALUE_REQUIRED, 'Specify root namespace')
```

Comment:
Could we instead not have the bundle-less option? What I mean is: allow the bundle option to be a bundle *or* a namespace. First, see if it’s a bundle. If it’s not, assume it’s the namespace. Then, why not just add an optional “path” argument, but throw an exception if it’s not passed if a namespace is used: require it in that case.

The final command would look like:

bin/console doctrine:mapping:import App annotation --path=src/

I think that’s simple enough. It’s a rare command: so just let them choose the namespace and path. For XML, the path we would recommend in the docs is config/doctrine

Reply:
@weaverryan I like your suggestion. @barell what do you think?